docker/run_update_data.sh

```diff
@@ -25,6 +25,7 @@ docker-compose run -u "${HOSTUSER}" crontabber ${CRONTABBERCMD} \
                --crontabber.class-FTPScraperCronApp.products=${PRODUCTS}
 
 # Update featured versions data based on release data
+docker-compose run crontabber ${CRONTABBERCMD} --reset-job=featured-versions-automatic
```

Comment:
Before running crontabber jobs like this, we need to reset them first. Otherwise they might decide not to run.